src/2019/components/Faq.js

```diff
@@ -105,7 +110,7 @@ class FAQ extends Component {
                                 </div>
                             </div>
                             <div className="text-center text-white w-100">
-                                Got any other questions? Email us at <a href="mailto:hello@sdhacks.io"> {' '} hello@sdhacks.io</a>! (Yep, we're affiliated with SD Hacks)
+                                Got any other questions? Email us at <FaqContactLink as="a" href="mailto:hello@sdhacks.io"> {' '} hello@sdhacks.io! </FaqContactLink>(Yep, we're affiliated with SD Hacks)
```

Comment:
you don't need the as="a" here because you already define `FaqContactLink` to be an `a` in line 31